collectors/ebpf.plugin/ebpf_socket.c

```diff
@@ -86,11 +85,13 @@ static void ebpf_update_global_publish(
         move = move->next;
     }
 
-    tcp->write = -((long)publish[0].nbyte);
-    tcp->read = (long)publish[1].nbyte;
+    // We read bytes from function arguments, but bandiwdth is given in bits,
+    // so we need to multiply by 8 the read data.
+    tcp->write = -(((long)publish[0].nbyte)<<3)/1000;
+    tcp->read = (((long)publish[1].nbyte)<<3)/1000;
 
-    udp->write = -((long)publish[3].nbyte);
-    udp->read = (long)publish[4].nbyte;
+    udp->write = -(((long)publish[3].nbyte)<<3)/1000;
+    udp->read = (((long)publish[4].nbyte)<<3)/1000;
```

Comment:
I used shift, because it is faster than multiply.

I am converting here, instead inside the function, because the function that writes dimension is also called to write a value that does not need adjusts from `ebpf_process.c`.

Reply:
You can pass the divisor to `ebpf_create_charts_on_apps` as a parameter.